### PR TITLE
Updated the readme file to remove the use of legacy app

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
#### Summary
1) Updated the readme file to remove the use of the legacy app as it has been depreciated by Zoom.
2) Updated all the other steps for the app and features to now according to the new zoom UI and also updated the screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,29 +61,23 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
 #### Create an app for Mattermost
 
 1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
-2. In the top left select **Develop** and then **Build Legacy App**.
-3. Select **OAuth** in **Choose your app type** section.
-4. Enter a name for your app.
+2. In the top left select **Develop** and then **Build App**.
```
Comment (mickmister) [inline README.md:64]:
```suggestion
2. In the top right select **Develop** and then **Build App**.
```

File: README.md [hunk 2]
```diff
@@ -61,29 +61,23 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
...
+3. On top you can choose either the **development** or **production** app. If you wish to publish the app on Zoom Marketplace then please select production. The plugin supports apps that are published in the Zoom Marketplace.
```
Comment (mickmister) [inline README.md:65]: I don't think the customer will be publishing it to the marketplace. I guess I don't see the use case of the customer publishing the app. Maybe we should just suggest select `Development` and explain why?

Also, is there a risk of choosing `Production`, even if they are not intending to publish to the marketplace? If it's not possible to set up with `Production`, and we can't do anything about that, then we need to say `Development`, but I would prefer to tell the customer to use `Production` if possible.

```suggestion
3. On top choose **Development**. We would choose **Production** if we were publishing to marketplace, but we won't be doing that here.
```

File: README.md [hunk 3]
```diff
@@ -61,29 +61,23 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
...
-![create_user_managed](https://github.com/mattermost/mattermost-plugin-zoom/assets/74422101/b42465a6-f91d-4737-9697-5636e75addf0)
-
-#### Configure your new OAuth app to work with Mattermost
 
-If you **would like to publish on Zoom Marketplace**, you'll find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.
+![app_type](https://github.com/mattermost/mattermost-plugin-zoom/assets/90389917/47b81f49-3699-4737-815d-5cc2fccab0c4)
 
-1. Go to the **App Credentials** tab on the left. Here you'll find your **Client ID** and **Client Secret**. These will be needed when you set up Mattermost to use the plugin.
+6. Next you'll find your **Client ID** and **Client Secret**. Please copy this as these will be needed when you set up Mattermost to use the plugin.
 2. Enter a valid **Redirect URL for OAuth** \(`https://SITEURL/plugins/zoom/oauth2/complete`\) and add the same URL under **Add Allow List**. Note that `SITEURL` should be your Mattermost server URL.
```
Comment (mickmister) [inline README.md:72]: Weird how the ordering goes from 6 to 2 here

File: README.md [hunk 4]
```diff
@@ -115,30 +109,20 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
...
-5. Choose **Account-level app** as the app type.
-6. Choose whether you **Would like to publish this app on Zoom Marketplace**. In most cases you'll want this to be disabled, but the plugin supports apps that are published in the Zoom Marketplace.
-7. Select **Create**.
-
-![Create account level app](https://github.com/mattermost/mattermost-plugin-zoom/assets/74422101/a7af4594-9b22-43ea-9dd6-c4bf7ecbd794)
-
-#### Configure your new OAuth app to work with Mattermost
-
-If you **would like to publish on Zoom Marketplace**, you'll find two sets of values: **development** and **production**. Make sure you follow the next steps with the production values.
+3. On top you can choose either the **development** or **production** app. If you wish to publish the app on Zoom Marketplace then please select production. The plugin supports apps that are published in the Zoom Marketplace.
```
Comment (mickmister) [inline README.md:112]: Same comment about publishing as before

File: README.md [hunk 5]
```diff
@@ -115,30 +109,20 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
...
 
-1. Go to the **App Credentials** tab on the left. Here you'll find your **Client ID** and **Client Secret**.
-   * These will be needed when you set up Mattermost to use the plugin.
-2. Enter a valid **Redirect URL for OAuth** \(`https://SITEURL/plugins/zoom/oauth2/complete`\) and add the same URL under **Add Allow List**.
-   * `SITEURL` should be your Mattermost server URL.
+![app_type](https://github.com/mattermost/mattermost-plugin-zoom/assets/90389917/3030b7ea-eea2-4fd1-b8f1-5b47a17efecc)
 
-![credentials](https://github.com/mattermost/mattermost-plugin-zoom/assets/55234496/587d6c25-f1c2-4744-b4ca-86f56ec9a6a5)
+6. Next you'll find your **Client ID** and **Client Secret**. Please copy this as these will be needed when you set up Mattermost to use the plugin.
+2. Enter a valid **Redirect URL for OAuth** \(`https://SITEURL/plugins/zoom/oauth2/complete`\) and add the same URL under **Add Allow List**. Note that `SITEURL` should be your Mattermost server URL.
```
Comment (mickmister) [inline README.md:119]: Same 6 to 2 comment here

File: README.md [hunk 6]
```diff
@@ -115,30 +109,20 @@ You can set the **OAuth ClientID** and **OAuth Secret**, generated by Zoom, and
 
 1. Go to [https://marketplace.zoom.us/](https://marketplace.zoom.us/) and log in as an admin.
 2. In the top left select **Develop** and then **Build App**.
```
Comment (mickmister) [inline README.md:111]:
```suggestion
2. In the top right select **Develop** and then **Build App**.
```

**Review by mickmister — COMMENTED** [review]
@Kshitij-Katiyar Thanks for making these updates :+1:  Just a few comments on wordings of things

**Comment by cwarnermm** [comment]
Migrating these updates to the Mattermost Product Documentation site via https://github.com/mattermost/docs/pull/7030.

**Pull request merged**